### PR TITLE
Improve integration test

### PR DESCRIPTION
**Pull request by odeac**
## Overview
add command line arguments for setting the timeouts for the tests
change the scope of slow fixtures to "package" in order to run the setup it only once per package

### Which JIRA issue does this PR relate to? If there is not a JIRA issue addressing this work, please create one now and add the link here.
Add link to corresponding JIRA issue.

### Complete this checklist before you submit the PR
- [x] This PR contains no more than 200 lines of code, excluding test code.
- [x] This PR meets [RChain development coding standards](https://rchain.atlassian.net/wiki/spaces/DOC/pages/28082177/Coding+Standards).
- [x] You have someone in mind to assign for review. Make this assignment after submitting the PR.

### Notes
Optional. Add any notes on caveats, approaches you tried that didn't work, or anything else.

### CODE REVIEW

File: integration-testing/test/conftest.py [hunk 1]
```diff
@@ -11,6 +11,12 @@ def pytest_addoption(parser):
     parser.addoption(
         "--peer-count", action="store", default="2", help="number of peers in the network (excluding bootstrap node)"
     )
+    parser.addoption(
+        "--start-timeout", action="store", default="0", help="timeout in seconds for starting a node. Defaults to 30 + peer_count * 10"
```
Comment (birchmd) [inline integration-testing/test/conftest.py]: Maybe make the default equal to 30 and remove the if statement below?

File: integration-testing/test/conftest.py [hunk 2]
```diff
@@ -11,6 +11,12 @@ def pytest_addoption(parser):
...
+    )
+    parser.addoption(
+        "--converge-timeout", action="store", default="0", help="timeout in seconds for network converge. Defaults to 200 + peer_count * 10"
```
Comment (birchmd) [inline integration-testing/test/conftest.py]: Same here

File: integration-testing/test/conftest.py (hunk 1 above)
Comment (odeac) [reply integration-testing/test/conftest.py]: if there is no value provided the formula `start-timeout = 30 + peer-count * 10` is used. If a value is provided then we use that exact value

File: integration-testing/test/conftest.py (hunk 2 above)
Comment (odeac) [reply integration-testing/test/conftest.py]: same here, with default formula `converge-timeout=200 + peer-count * 10`

**Pull request merged**